.github/workflows/nightly.yml

```diff
@@ -11,14 +11,18 @@ jobs:
       should_run: ${{ steps.should_run.outputs.should_run }}
     steps:
       - uses: actions/checkout@v2
+
+      - name: current Boston time
```

Comment:
is boston really the city of note
:NYC:

Reply:
Boston is the center of the Algorand universe.